easy-ui-tokens/src/shape/base.json

```diff
@@ -1,7 +1,8 @@
 {
   "shape": {
     "border_radius": {
-      "sm": { "value": "3px", "attributes": { "unit": "rem" } },
+      "xs": { "value": "3px", "attributes": { "unit": "rem" } },
```

Comment:
i guess this is technically a breaking change since there is at least one place where this is being consumed outside this project that would need to be updated. that appears to be the `Search` component from the docs site which uses `sm`. 

im open to other ways to handle this (assuming the `4px` from the Figma file is intentional)

Reply:
good question. our border radiuses today feel a little arbitrary. it might be good to ask Rachel if there are a set of border radiuses that she uses in the designs/would like to systematize, and we can hopefully adjust once and for all

Reply:
updated `border_radius` mappings per discussion with rachel. should be documented now in Figma.